docs/source/markdown/options/secret.md

```diff
@@ -19,8 +19,13 @@ Secrets and its storage are managed using the `podman secret` command.
 
 Secret Options
 
-- `type=mount|env`    : How the secret will be exposed to the container. Default mount.
-- `target=target`     : Target of secret. Defaults to secret name.
+- `type=mount|env`    : How the secret will be exposed to the container. Defaults to `mount`.
+- `target=target`     : Target of secret. 
```

Comment:
Maybe add an example showing this in action?

Reply:
Added an examples section!